model/report_sources/proof_of_concept_2020-03-22.Rmd

```diff
@@ -43,13 +51,14 @@ admissions. In short, the process is as follows:
 1. use a recent number of admissions at a given date as a point of reference
 
 2. simulate growth in admissions from this point of reference using a given
-   doubling time, feeding into a log-linear model, including user-specified
-   upper and lower bounds
+   doubling time (with user-specified upper and lower bounds), feeding into a
+   log-linear model
    
 3. for each simulated admission, simulate the duration of hospitalisation from
    recent estimate of the duration of hospitalisation; it may be required to
-   repeat this process many time for lower numbers of admission, to integrate

```

Comment:
repeat this process many times for lower numbers of admissions

Reply:
been reformulated